### ISSUE TITLE
Jump to object not supported in your NW release

### ISSUE THREAD

**Issue by AndreaBorgia-Abo** [abapGit/abapGit#6028]
Ran ATC checks from "Advanced" menu, got a couple of errors, wanted to check them via click navigation, got the error above.
This is on a 731 System, but the click to navigate function in other places seems (main repo list, for example) seems to work just fine.
CL_CI_TESTS exists and has a GET_TEST_REF method, so [the call ](https://github.com/abapGit/abapGit/blob/ca02b3d4aa84db61e0fa40cb4e52d936ac17aadc/src/ui/pages/zcl_abapgit_gui_page_codi_base.clas.abap#L151) should work just fine.
![grafik](https://user-images.githubusercontent.com/25269970/216334246-271e91a4-b0c6-4797-a023-d38da63caa7e.png)
(CL_CI_TEST_SYNTAX_CHECK also exists)

**Comment by larshp** [comment]
try checking what the exception is?

**Comment by larshp** [comment]
yea, so either P_TEST or P_RESULT changed types during SAP versions

**Comment by AndreaBorgia-Abo** [comment]
I guess this earns the issue a wontfix tag :(
I tried using ADT as navigation but this option is blocked on my laptop, I asked the admin to look into it.
